docs/user_guide/assessment.rst

```diff
@@ -407,12 +407,55 @@ to take advantage of the inherent plotting capabilities of
 
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
-    :start-after: # Analyze metrics using MetricFrame
+    :lines: 32-48
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
     :target: auto_examples/plot_quickstart.html
     :align: center 
 
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: `ylim`
+'''''''''''''''''''''''
+The y-axis range is automatically set, which can be misleading, therefore it is
+sometimes useful to set the `ylim` argument to define the yaxis range.
```

Comment:
nit: can you use `:code:` before `ylim`? That way it should render as code rather than in italics. [Hopefully I'm not mixing things up, but feel free to tell me I'm wrong!]

Reply:
same for  other code occurrences below, of course 😄

Reply:
cool! didn't notice the diff between italics and code, changed!